include/zephyr/drivers/flash.h

```diff
@@ -138,6 +140,7 @@ __subsystem struct flash_driver_api {
 	flash_api_sfdp_read sfdp_read;
 	flash_api_read_jedec_id read_jedec_id;
 #endif /* CONFIG_FLASH_JESD216_API */
+	flash_api_ex_op ex_op;
```

Comment:
Add `CONFIG_FLASH_HAS_EX_OP` Kconfig options, which should be selected by drivers supporting `ex_op`.
Then make existence of `ex_op` dependent on the `CONFIG_FLASH_EX_OP_ENABLED`, which in would be Kconfig options appearing only when any driver selects `CONFIG_FLASH_HAS_EX_OP`.

Reply:
Done